### DISCUSSION TITLE
Welcome to Environnemental-Sensing Discussions!

### DISCUSSION THREAD

**Discussion by loco-philippe** [loco-philippe/Environmental-Sensing#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by loco-philippe** [comment]
The Environmental-sensing project was born from the following observations:

- there is no standard format (apart from the Environmental Sensing Service Bluetooth) used by the sensors to transmit the information (binary and textual),
- there is no data exchange format presenting at the same time a temporal, spatial and physical component (apart from file formats),
- we spend a lot of energy converting this type of data to make it usable.
- few tools are available to analyze the relationship between lists of values (like complex csv file)

The objective of the project is therefore to improve the interoperability of environmental data through the convergence of standards and the sharing of libraries facilitating the use of these standards and the accessibility of data.

if you are interested in this project, I will be very happy to discuss 
- questions you have, 
- ideas you want to share, 
- your involvement and your participation

Welcome to this community !

**Comment by martin1d544** [comment]
Hello,
I am an engineering student on a gap year and I started working on this project as part of the Bluehats Semester of Code.
I am currently working on a tool to write MongoDB queries suitable for the observation format.
